### ISSUE TITLE
Submodules vs clone in Unity projects

### ISSUE THREAD

**Issue by deadlyfingers** [Unity3dAzure/UnityWebSocket#1]
TLDR: Removing git submodules in favour of clone / manual copying.

Initially I had thought to use git submodules to handle installing dependencies with this repo. But as these scripts are designed to be dropped onto Unity game objects directly I was getting broken links when cloning a new copy of a Unity project. Therefore I will simply keep this repo as a stripped down version and link to any dependencies in the README which should be cloned / manually copied separately into your Unity project. 

Bottom line: When you have scenes with scripts linked to game objects then it works better if the scripts are bundled with the Unity project.
